src/components/menu/Menu.md

```diff
@@ -28,7 +27,7 @@ const MenuSectionHeader = require('box-ui-elements/es/components/menu')
   <MenuSeparator />
   <MenuSectionHeader>Menu Section</MenuSectionHeader>
   <MenuLinkItem>
-    <Link href="/#">Awesome Link</Link>
```

Comment:
nit: looks like you can keep this as `<Link />`s in this file and `SelectMenuLinkItem.md`

Reply:
fixed